search-data.json

```diff
@@ -18,8 +18,7 @@ Exclude some pages that we don't want to index.
   page.url == '/packages/pulumi-kubernetes/' or
   page.url == '/redirects.json' or
   page.url == '/search-data.json' or
-  page.url == '/search.html' or
-  page.url == '/welcome.html' -%}
+  page.url == '/search.html' -%}
```

Comment:
Should this be in the current PR, or is this a relic from a previous commit on your machine?

Reply:
This PR deletes `/welcome.html`, so it’s no longer needed here.